crates/rooch-framework/sources/bcs_friend.move

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+/// Source from https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-stdlib/sources/from_bcs.move
+
+/// This module provides a number of functions to convert _primitive_ types from their representation in `std::bcs`
+/// to values. This is the opposite of `bcs::to_bytes`. Note that it is not safe to define a generic public `from_bytes`
+/// function because this can violate implicit struct invariants, therefore only primitive types are offerred. If
+/// a general conversion back-and-force is needed, consider the `moveos_std::Any` type which preserves invariants.
+module rooch_framework::bcs_friend {
```

Comment:
We do not need to add a new module for this, we just need a function. And the rooch_framework do not need a bcs module.

Reply:
This PR has a lot of files, I have already addressed these issues in another PR.